FeatureFlags.md

```diff
@@ -1,27 +1,4 @@
-- `graphql-docs-generator` - generates GraphQL operations from the given GraphQL schema. Accessed via CLI command `amplify codegen statements` and `amplify-codegen`
-- `graphql-types-generator` - generates type definitions in given target language for the types defined in the GraphQL schema. Accessed via CLI command `amplify codegen types` and `amplify-codegen`
+To find out more about supported feature flags, refer to the [Amplify CLI Feature Flag Documentation](https://docs.amplify.aws/cli/reference/feature-flags/#codegen).
 
-are transferred to this repository from the [Amplify CLI repository](https://github.com/aws-amplify/amplify-cli/tree/master/packages). 
-
-In order to shield the existing customers from any complications as a result of this transfer, we provide the following Feature Flags that enable the customers to switch to the migrated package implementations at their own pace before the deprecation date:
-
-1. **useAppsyncModelgenPlugin**: When set to `true`, the `appsync-modelgen-plugin` implementation in this repository will be used for models generation. When set to `false`, the `amplify-codegen-appsync-model-plugin` implementation from [CLI repo](https://github.com/aws-amplify/amplify-cli/tree/master/packages/amplify-codegen-appsync-model-plugin) will be used for models generation. 
-
-2. **useDocsGeneratorPlugin**: When set to `true`, the `graphql-docs-generator` implementation in this repository will be used to generate the GraphQL operations. When set to `false`, the `amplify-graphql-docs-generator` implementation from [CLI repo](https://github.com/aws-amplify/amplify-cli/tree/master/packages/amplify-graphql-docs-generator) will be used to generate the GraphQL operations. 
-
-3. **useTypesGeneratorPlugin**: When set to `true`, the `graphql-types-generator` implementation in this repository will be used to generate the language specific type definitions from GraphQL schema. When set to `false`, the `amplify-graphql-docs-generator` implementation from [CLI repo](https://github.com/aws-amplify/amplify-cli/tree/master/packages/amplify-graphql-types-generator) will be used to generate the type defintions. 
-
-The package implementations in this repository will be actively maintained going forward while the implementations in [Amplify CLI repository](https://github.com/aws-amplify/amplify-cli/tree/master/packages) will be deprecated on May 1st, 2021. 
-The package implementations in this repository will support all the APIs that are supported by their counterparts in the CLI repository. 
-
-These Feature Flags are set to `true` by default for new projects created using Amplify CLI and `false` for existing projects.
-We recommend setting the above Feature Flags to `true` in your `cli.json` file to consume the package implementations from this repository and take advtange of bug fixes, enhancements etc being made to these generators.
```

Comment:
This might be a useful info to retain as it conveys to the developer that they should work on setting these FFs to `true`. We might additionally also want to callout that each FF has a deprecation date set to give them time to migrate.

Reply:
> Another smoother approach we could take is to not release these code changes yet but first set these FFs to true for all projects in CLI, see the customer response and then go ahead with this release. This would be easier to roll back just in case.

That probably makes some sense. We can drop it and revisit it in a month or two.

I think I'll take that approach, and just go with the CLI/docs update.